src/editmap.cpp

```diff
@@ -1973,7 +1931,7 @@ int editmap::edit_mapgen()
         gmenu.addentry( -1, !id.id().is_null(), 0, "[%3d] %s", static_cast<int>( id ), id.id().c_str() );
         gmenu.entries[i].extratxt.left = 1;
         gmenu.entries[i].extratxt.color = id->get_color();
-        gmenu.entries[i].extratxt.txt = string_format( "%c", id->get_symbol() );
+        gmenu.entries[i].extratxt.txt = id->get_symbol();
```

Comment:
Good catch - that's was definitely an error. I think this is from RARELY used debug menu and that's is why it wasn't caught earlier.

Broken:

![image](https://user-images.githubusercontent.com/16213433/56855355-553df480-694e-11e9-8e6d-8b39e174f4b6.png)


Fixed:

![image](https://user-images.githubusercontent.com/16213433/56855331-1f007500-694e-11e9-9917-caba1485327f.png)